lib/style_parser/index.js

```diff
@@ -20,7 +20,8 @@ function parse(cssInput, prefix, postfix, cb) {
   async.waterfall([
     // remove everything except style tag contents
     function(cb) {
-      sanitizer.sanitize(cssInput, cb);
+      var requireStyleTags = true;
```

Comment:
What if we added the `<style>` tags in this file instead of in the sanitizer? Then we wouldn't have to pass that option in and gives a more specific focus to the style sanitizer.